java/client/src/test/java/glide/api/RedisClientTest.java

```diff
@@ -4286,6 +4307,7 @@ public void bitcount_returns_success() {
 
         // verify
         assertEquals(testResponse, response);
+        assertEquals(1L, payload);
```

Comment:
nit: I think this line and the line immediately below are essentially the same, since bitcount holds the value of 1L, so I think we do not need to add this extra line

Reply:
i will remove it